apps/explorer/src/ui/Tabs.tsx

```diff
@@ -22,41 +16,21 @@ export function TabPanel({ noGap = false, ...props }: TabPanelProps) {
     return <HeadlessTab.Panel className={noGap ? '' : 'my-4'} {...props} />;
 }
 
-export type TabGroupProps = ExtractProps<typeof HeadlessTab.Group> & {
-    size?: TabSize;
-};
+export type TabGroupProps = ExtractProps<typeof HeadlessTab.Group>;
 
-export function TabGroup({ size, ...props }: TabGroupProps) {
-    return (
-        <TabSizeContext.Provider value={size}>
-            <HeadlessTab.Group as="div" {...props} />
-        </TabSizeContext.Provider>
-    );
+export function TabGroup(props: TabGroupProps) {
+    return <HeadlessTab.Group as="div" {...props} />;
 }
 
-const tabStyles = cva(
-    [
-        'border-b border-transparent ui-selected:border-gray-65 font-semibold text-steel-dark hover:text-steel-darker active:text-steel pb-2 -mb-px',
-    ],
-    {
-        variants: {
-            size: {
-                lg: 'text-heading4 ui-selected:text-steel-darker',
-                md: 'text-body ui-selected:text-steel-darker',
-            },
-        },
-        defaultVariants: {
-            size: 'md',
-        },
-    }
-);
-
 export type TabProps = ExtractProps<typeof HeadlessTab>;
 
 export function Tab({ ...props }: TabProps) {
-    const size = useContext(TabSizeContext);
-
-    return <HeadlessTab className={tabStyles({ size })} {...props} />;
+    return (
+        <HeadlessTab
+            className="-mb-px border-b border-transparent pb-2 text-body font-semibold text-steel-dark hover:text-steel-darker active:text-steel ui-selected:border-gray-65 ui-selected:text-steel-darker lg:text-heading4"
```

Comment:
This isn't quite right. We still have two tab size variants, each of which should have its own responsive treatment.

Reply:
We need the smaller tabs for the mobile viewport here https://www.figma.com/file/kSD0qXCIDZA4KQVNepA5hR/02-Explorer?node-id=2-9&t=2ikLV3SREWuoDkCQ-0. They are basically going from `lg` to `md` after adjusted.

Unless we are using `useBreakpoint` to track it, otherwise we'd be going from `lg` to `md` and `md` to `sm`, which is then too small to really use it.